kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -158,7 +167,8 @@
     },
     methods: {
       isActiveLink(route) {
-        return `${this.link}#${route}` === `${window.location.pathname}${window.location.hash}`;
+        const full_path = !route.includes('#') ? `${this.link}#${route}` : route;
```

Comment:
👀 . maybe `fullPath` instead? 🙂

Reply:
I am wondering if this method is even needed any more? The link will only be active when we are in the single page app itself, in which case, we will be using KRouterLink, which will handle the bold styling internally. This means that we don't need to conditionalize the styling using `isActiveLink` and we could probably get rid of the method completely.

Reply:
@rtibbles @marcellamaki, this seems like an important piece to the stying from what I can remember. @AllanOXDi?

Reply:
It seems if this is not the case, this is a flaw in the KRouterLink implementation, as the base <router-link> component has support to allow `isActive` and isExactActive` to be propagated into slots included in it: https://v3.router.vuejs.org/api/#example-applying-active-class-to-outer-element

My main concern here is us expending time and energy trying to get the 'isActiveLink' algorithm correct, which would be better deferred to <router-link> which has already solved this problem.

Reply:
That said - if it would be better to merge this and open a follow up issue, I am fine with that - we just have to be open to the possibility that there will still be bugs in the styling because of our hand made isActiveLink checking.